src/model/PartyDetail.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export interface PartyDetail {
```

Comment:
The classes under `model` use camel case rather then underscore, this should adopt the same convention.

Reply:
This is called `PartyDetails` in the underlying ProtoBuffer definition, keep it as is, please.